src/annotator/index.js

```diff
@@ -61,7 +61,9 @@ function init() {
 
     const sidebarConfig = getConfig('sidebar');
 
-    const hypothesisAppsOrigin = new URL(sidebarConfig.sidebarAppUrl).origin;
+    const hypothesisAppsOrigin = new URL(
+      /** @type {string} */ (sidebarConfig.sidebarAppUrl)
+    ).origin;
```

Comment:
`sidebarConfig` is now a `Record<string, unknown>` so I'm adding a cast here to say "trust me, this is a string".